cmd/metal-api/internal/service/network-service.go

```diff
@@ -409,6 +417,28 @@ func (r *networkResource) createNetwork(request *restful.Request, response *rest
 	r.send(request, response, http.StatusCreated, v1.NewNetworkResponse(nw, usage))
 }
 
+func validateAdditionalAnnouncableCIDRs(additionalCidrs []string, privateSuper bool) ([]string, error) {
```

Comment:
Maybe more like this?

```suggestion
func validateAdditionalAnnouncableCIDRs(additionalCidrs []string, privateSuper bool) error {
```